model/src/main/java/com/basistech/rosette/apimodel/CategoriesOptions.java

```diff
@@ -31,8 +31,24 @@
 public class CategoriesOptions extends Options {
 
     /**
+     * @deprecated
+     * users should use {@link singleLabel} to return only one result
+     * or {@link scoreThreshold} to filter results based on raw score
      * @return number of categories
      */
     @Min(1)
     private final Integer numCategories;
+
+    /**
+     * Single label mode will return only the highest scoring category
+     * @return whether or not we are in single label mode
+     */
+    private final boolean singleLabel;
```

Comment:
Let's box this per our convention. If both this and `threshold` are set, who wins?

Reply:
`threshold` wins with respect to the threshold value itself, but we will still only return a maximum of one result if both are set